### ISSUE TITLE
[TB, Epic] Create better documentation linking WCAG, atom and molecule values

### ISSUE THREAD

**Issue by aaronreed708** [finos/a11y-theme-builder#750]
## Problem/Concern

Consider this as an epic, whose goal is to help users appreciate how the values they select in atoms and molecules will affect the overall accessibility of the resulting design system.  Another goal is to educate the user on how the WCAG standards affect the choices that they available to them and why.

Tasks governed by this epic:
- [ ] Write documentation around how shades, chroma, etc affect accessibility (#751)
    - [ ] Identify what, if any, of this new documentation should be available to the user through the UI
- [x] Design Thinking on how a user might best discover the information through the UI (#752)
- [ ] Identify and link relevant WCAG guidelines to appropriate atoms and molecules. Write documentation to be displayed to the user (#754)
- [ ] Update UI with documentation per design produced by Design Thinking (#753)

**Comment by lwnoble** [comment]
See Figma File: https://www.figma.com/file/qplg5PqoDAK6dh9Y58JlMD/Building-Themes?type=design&mode=design&t=PUwaunPdRh9LHxLm-1

**Comment by lwnoble** [comment]
The WCAG requirements for each component:
https://hedgedoc-hedgedoc.apps.aws-useast1-apps-dev-1.ocpdev.us-east-1.ac.discoverfinancial.com/9zB8IpsbRcyirC4Js-e4LA?both
